src/router.ts

```diff
@@ -4,12 +4,14 @@ import * as user from './requestHandlers/user';
 import * as hook from './requestHandlers/hook';
 import * as panel from './requestHandlers/panel';
 import * as panelSet from './requestHandlers/panelSet';
+import * as utils from './requestHandlers/utils';
 
 /**
  * Route all incoming requests
  */
 
 export default (app: Express) => {
+    app.get('*', utils.notFound)
```

Comment:
![image](https://github.com/RIT-Crowd-Comic/crowd-comic-back-end/assets/89936632/2e1c4ccc-889c-45a1-88db-71414afdde39)

I am getting a bug where at least the 3 GET queries for panel don't work if this line isn't commented out. It could be I did something wrong, as other queries like createUser or panel still work.

Reply:
fixed